javascript/net/grpc/web/grpcwebclientreadablestream.js

```diff
@@ -136,9 +136,8 @@ class GrpcWebClientReadableStream {
 
     const self = this;
     events.listen(this.xhr_, EventType.READY_STATE_CHANGE, function(e) {
-      let contentType = self.xhr_.getStreamingResponseHeader('Content-Type');
-      if (!contentType) return;
-      contentType = contentType.toLowerCase();
+      const contentType = (self.xhr_.getStreamingResponseHeader('Content-Type') || '').toLowerCase();
```

Comment:
Could you explain a bit more on the motivation of all these changes (besides just adding the `httpStatusCode` in metadata)?

It looks to me that it's altering some existing logic, and I'm not sure if it can cause regressions.

From the PR description, I got the impression that adding `httpStatusCode` was all we needed.

Is that not the case?

Reply:
Hi, already described it above: https://github.com/grpc/grpc-web/pull/1472#issuecomment-2636917816

Reply:
@sampajano can we move this forward?

Reply:
@wszydlak Hey sorry i was caught up with other business.. I'll try to get back to this early next week!

Reply:
@sampajano any updates?

Reply:
@sampajano can we move this PR forward?

Reply:
@sampajano are you there? Can we do something with this PR?

Reply:
@wszydlak Hi Wotjek. My deep apologies for long delays on this PR. I was caught up with some personal issues and other work priorities, so had been procrastinating on this. Sorry for the delays! I'll take another look now!